hummingbot/cli/utils/stop_loss_tracker.py

```diff
@@ -55,11 +55,15 @@ def get_balances(self) -> Dict[str, float]:
 
     async def stop_loss_loop(self):
         while True:
-            self._current_pnl = self.calculate_profit_loss_pct(self._stop_loss_price_type)
-            if self._current_pnl <= -self._stop_loss_pct:
-                self._stop_handler()
-            else:
-                await asyncio.sleep(self._update_interval)
+            try:
+                self._current_pnl = self.calculate_profit_loss_pct(self._stop_loss_price_type)
+                if self._current_pnl <= -self._stop_loss_pct:
+                    self._stop_handler()
+                    break
```

Comment:
Please follow the usual `try.. except..` convention (even though it's troublesome). This one has another problem because it's now not cancellable.

It should be

```
  try:
    (do something)
  except asyncio.CancelledError:
    raise
  except Exception:
    self.logger().error(..., exc_info=True)

  await asyncio.sleep(...)
```

Reply:
done